lib/lint.ml

```diff
@@ -260,6 +260,14 @@ module Checks = struct
         if is_build then [ (pkg, DuneIsBuild) ] else errors
     | None -> []
 
+  let check_maintainer_email ~pkg opam =
+    let maintainers = OpamFile.OPAM.maintainer opam in
+    List.filter_map
+      (fun m ->
+        if Str.string_match (Str.regexp ".*<?.*@.*>?") m 0 then None
```

Comment:
There are some packages in the opam-repository, which have just an email address in the maintainer field. This check allows for that, instead of requiring a "Name <email>" format.